tests/wpt/mozilla/tests/mozilla/mozbrowser/mozbrowser_click_fires_openwindow.html

```diff
@@ -0,0 +1,26 @@
+<head>
+<title>Browser API; mozbrowseropenwindow event</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+<script>
+
+async_test(function(t) {
+
+  var iframe = document.createElement("iframe");
+  iframe.mozbrowser = "true";
+  iframe.src = "mozbrowser_click_fires_openwindow_iframe1.html";
+
+  iframe.addEventListener("mozbrowseropenwindow", t.step_func(function(e) {
+    assert_equals(e.detail.target, "_blank");
+    assert_equals(e.detail.url.split("/").pop(), "mozbrowser_click_fires_openwindow_iframe3.html");
```

Comment:
Here, I think you want to make sure the URL has been properly joined. This will do it:
```js
var expected_url = new URL("mozbrowser_click_fires_openwindow_iframe3.html", window.location.href);
assert_equals(e.detail.url, expected_url);
```